acoustid/web/views/apps.py

```
@@ -23,36 +24,40 @@
 def applications():
     user = require_user()
     title = "Your Applications"
-    applications = find_applications_by_account(db.session.connection(), user.id)
+    applications = find_applications_by_account(db.get_app_db(), user.id)
     return render_template("applications.html", title=title, applications=applications)
 
 
 @apps_page.route("/application/<application_id>")
 def application(application_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `application`

(ANN201)